lib/src/tasks/coverage/api.dart

```diff
@@ -214,13 +214,45 @@ class CoverageTask extends Task {
 
   Future _collect() async {
     List<File> collections = [];
+    PubServeTask pubServeTask;
+    PubServeInfo serveInfo;
+
+    if (pubServe && pubServeTask == null) {
```

Comment:
`pubServeTask` will always be null here since it is defined locally immediately before this